### DISCUSSION TITLE
How to add NOTE field to checkout (in createCheckout or in other mutation)

### DISCUSSION THREAD

**Discussion by DidoMarchet** [saleor/saleor#9580] [General] [answered]
Hi guys,
we need to add gift note to our checkout process.
I didn't find a mutation that let me to add Note field.

I tried with CreateCheckout too but the argument `input: CheckoutCreateInput!` doesn't have Note in the type detail.

We released a new ecommerce and they ask firmly this feature.

Can you give me some advice please? Any suggestion is precious.

Thanks and regards,

Davide

**Comment by korycins** [comment, marked as answer]
Hey @DidoMarchet,
Looks like the mutation for adding a note to checkout is missing. It should be something like `checkoutAddNote`. Can you create a GH issue for this? 
As a temporary workaround, you could use checkout.metadata.

**Reply by DidoMarchet** [reply]
Hi,
thanks @korycins.
This is the issue [https://github.com/saleor/saleor/issues/9591](https://github.com/saleor/saleor/issues/9591)

regards,

Davide

**Comment by maarcingebala** [comment]
Fixed in https://github.com/saleor/saleor/pull/16315.
